Updated IS 6.0.0 migration docs (#3287)
wso2/docs-is: en/docs/deploy/migrate/migrate-to-600.md

    Make sure to point the internal keystore to the keystore that is copied from the previous WSO2 Identity Server version. The primary keystore can be pointed to a keystore with a certificate that has a strong RSA key.

>     Also make sure to add the public key of the primary keystore to the client trust store copied from the old IS

Reviewer: ```suggestion
    Also, make sure to add the public key of the primary keystore to the client trust store copied from the previous IS.
```

If you have created secondary user stores in the previous WSO2 IS version, copy the content in the `<OLD_IS_HOME>/repository/deployment/server/userstores` folder to the `<NEW_IS_HOME>/repository/deployment/server/userstores` folder.

!!! note

>     If you are migrating from a version prior to IS 5.5.0, you need to make the following changes in the `<NEW_IS_HOME>/migration-resources/migration-config.yaml` file. See Step 2 for instructions for downloading migration resources

Reviewer: ```suggestion
    If you are migrating from a version prior to IS 5.5.0, you need to make the following changes in the `<NEW_IS_HOME>/migration-resources/migration-config.yaml` file (see Step 2 for instructions for downloading migration resources).
```

>     - Remove all `UserStorePasswordMigrators` from versions above your old IS version.

Reviewer: ```suggestion
    - Remove all `UserStorePasswordMigrators` from versions above your previous IS version. Userstore password migration will be done by the `EncryptionAdminFlowMigrator` in the version 5.11.0.
```
